qemu/tests/win_heavyload.py

```diff
@@ -58,6 +58,19 @@ def add_option(cmd, key, val):
             cmd += " /%s %s " % (key, val)
         return cmd
 
+    def cleanup_temp(session):
+        """
+        del temp file that created by heavyload in VM
+
+        :param session: VM session
+        """
+        check_temp_cmd = 'dir %temp% %tmp% /s /b | findstr jam*'
+        s, o = session.cmd_status_output(check_temp_cmd)
+        if not s:
+            file_list = list(set(o.strip().splitlines()))
+            for k in file_list:
```

Comment:
I'd prefer to use a more proper variable name instead of `k` or `o`.

Reply:
s, o means status, ouput, it is a common usage.  i think it is no need to change.
About "k",  i think it is ok for iteration.  thanks for your review.

Reply:
Better not use `k` as in most cases, it's short for `key`

Reply:
thanks, updated.